config/locales/en.yml

```diff
@@ -475,22 +478,22 @@ en:
       features: "Posting features"
       tags: "#tags"
       tags_ex1: "Place a # (hash) symbol in front of a word and it becomes a #tag. What’s the use of this? Well, #tags are a great way of finding content that interests you, and of bringing content to the attention of people who are likely to be interested in it. You can search for tags by typing them in the search field in the header. You can also follow tags, so that any posts containing those tags appear in your stream – in fact, if you filled out the “Describe yourself in 5 words” part of your profile page, you’re already following those tags!"
       tags_ex2: "Say you like cooking and you post a message with “#cooking” in it. Anyone who follows the #cooking tag will now see your post appear in their streams. Of course for any follower of #cooking to see your post, you’ll have had to make it a public post; if you made it only to one or more aspects, only people in those aspects will be able to read it. You can also search for “#cooking” in the search bar, and you’ll be presented with a list of all posts tagged with “#cooking.” That includes your own posts, posts by friends and public posts by other diasporans. You can see how this can be a fantastic tool for finding and sharing content with people who share your interests."
+      nsfw: "The “Not safe for work” tag"
       tags_ex3: "If your post might potentially cause offence or get someone into trouble for viewing it at work, please add the <span class=\"click\">#nsfw</span> (“not safe for work”) tag so that it will be hidden on people’s screens unless they choose to view it. We’ll cover this in %{part7_link}."
       mentions: "@mentions"
       mentions_ex1: "Did you know you can grab someone’s attention by “mentioning” them? Simply type a @ symbol followed by the name of one of your contacts. As you start typing their name (their screen name, not their user name) it will automatically be completed by diaspora*. Hit enter or click the name in the auto-completer and you’ll notice that it has changed to their full name, and the @ symbol has disappeared. Don’t worry, though; this will become an @mention once you have posted the message. The person you are mentioning will receive a notification of the mention on their notification page, and by email if they have asked to receive email notification of mentions."
-      mentions_ex2: "Likewise other people can @mention you, and you’ll receive a notification in the header bar (and by email if you have set that option)."
-      mentions_ex3: "Note that you can only @mention people in your aspects; and only in posts, not in comments."
+      mentions_ex2: "You can @mention people in posts, comments and in private conversations (see %{part6_link}). Likewise other people can @mention you, and you’ll receive a notification in the header bar (and by email if you have set that option)."
```

Comment:
Oh, I definitely read somewhere in all the information coming out at the moment that mentions, along with the markdown publisher, are in conversations too. 

I couldn't actually see the point of that addition, so it makes sense that that isn't the case. I'll change it now. I'll also look through the various announcements to remove it if I can find where I read it. Thanks.